src/components/LinkToPage/LinkToPage.tsx

```diff
@@ -8,8 +8,6 @@ interface LinkTo {
 }
 
 const LinkToPage = (props: LinkTo) => {
```

Comment:
Is 'urlTo' a required prop for sure? If yes the ternaty operator in 15 line will alwas cast to true and styles.active will be added.